inst/lintr/linters.csv

```diff
@@ -14,6 +14,7 @@ comparison_negation_linter,readability consistency
 condition_message_linter,best_practices consistency
 conjunct_test_linter,package_development best_practices readability configurable pkg_testthat
 consecutive_assertion_linter,style readability consistency
+consecutive_mutate_linter,consistency readability configurable
```

Comment:
Maybe efficiency?

Reply:
These days I'm mostly using {dbplyr} backend so I forget about the efficiency part :)